src/signals/incident-management/containers/IncidentOverviewPage/index.test.js

```diff
@@ -243,10 +243,10 @@ describe('signals/incident-management/containers/IncidentOverviewPage', () => {
     expect(props.pageChangedAction).toHaveBeenCalledWith(pagenum);
   });
 
-  it('should render a map', () => {
+  it('should render a map', async () => {
```

Comment:
this is more of a integration than an unit test :-)

Reply:
Not really, it tests that specific parts are rendered in this component
🤷‍♂